web/packages/teleterm/src/ui/StatusBar/ShareFeedback/ShareFeedback.tsx

```diff
@@ -47,7 +47,7 @@ export function ShareFeedback() {
         onClick={openShareFeedback}
       >
         {!hasBeenShareFeedbackOpened && <NotOpenedYetIndicator />}
-        <ChatBubble fontSize="14px" />
+        <ChatBubble size="medium" />
```

Comment:
This icon is in the bottom right of the window. idk if it was always like that, but in the new design the icon looks misaligned.

Reply:
added a small margin below it, how's this?

### Before

![image](https://github.com/gravitational/teleport/assets/56373201/3ef7669b-a80f-4747-b149-5047f1db3dcd)

### After

![image](https://github.com/gravitational/teleport/assets/56373201/296a9ba4-ae89-420c-ac58-4bbafa8ced53)